styles/app.css

```diff
@@ -0,0 +1,89 @@
+.register-textbox {
+  height: 36px;
+  margin-bottom: 15px;
+  padding-left: 10px;
+  padding-right: 10px;
+  width: 394px;
+}
+
+.register-textbox.narrow {
+  margin-right: 10px;
+  width: 192px;
+}
+
+
+button, input, optgroup, select, textarea {
+  border-radius: 4px;
+  color: inherit;
+  font: inherit;
+  line-height: 1.42857143;
+  margin: 0;
+  padding: 2px 8px;
+}
+
+table td, table td * {
+  vertical-align: top;
+}
+
+.feedback-panel {
```

Comment:
use tailwind for all the styling. USe custom CSS only if not possible to di using tailwind